src/components/NcButton/NcButton.vue

```diff
@@ -321,24 +321,32 @@ export default {
 ```
 </docs>
 
-<script>
+<script lang="ts">
 import isSlotPopulated from '../../utils/isSlotPopulated.js'
```

Comment:
```suggestion
import isSlotPopulated from '../../utils/isSlotPopulated'
```

Reply:
Well, I don't know, I just saw the error message from jest
```
 Cannot find module '../../utils/isSlotPopulated.js' from 'src/components/NcButton/NcButton.vue'
```

Reply:
Good point, that needs to be fixed

Reply:
Solved :heavy_check_mark: